Tests/FrameworkTests/Card Brand Tests/CardBrandTest.swift

```diff
@@ -19,7 +19,7 @@ class CardBrandTest: XCTestCase {
         
         let config = VGSConfiguration(collector: storage, fieldName: "cardNum")
         config.type = .cardNumber
-        config.formatPattern = nil
+        config.formatPattern = ""
```

Comment:
Is it important to set an empty string than `nil`?

Reply:
nil - default mask, empty string - no mask.